### PR TITLE
[REVIEW] cybert streamz memory optimization

### PR DESCRIPTION
**Pull request by brhodes10**
Optimizing cybert streamz by 
- deleting torch variables in cybert inference function
- clearing torch cache in the streamz workflow 
- running garbage collection in the streamz workflow

### CODE REVIEW

**Comment by GPUtester** [comment]
Please update the changelog in order to start CI tests.

View the gpuCI docs [here](https://docs.rapids.ai/gpuci#open-pr-or-update-pr).

**Comment by brhodes10** [comment]
rerun tests

File: python/clx/analytics/cybert.py [hunk 1]
```diff
@@ -69,7 +69,7 @@ def load_model(
         """
         with open(config_filepath) as f:
             config = json.load(f)
-        self._label_map = {int(k): v for k, v in config['id2label'].items()}
+        self._label_map = {int(k): v for k, v in config["id2label"].items()}
```
Comment (BartleyR) [inline python/clx/analytics/cybert.py:72]: This label map is typically small, correct? I'm just wondering about the loop here. If it's small though, no need to optimize now.

**Review by BartleyR — COMMENTED** [review]
Approved, but have a question about one of the label map portions.

File: python/clx/analytics/cybert.py (hunk 1 above)
Comment (brhodes10) [reply python/clx/analytics/cybert.py:72]: Yes it is small cc @raykallen

Comment (BartleyR) [reply python/clx/analytics/cybert.py:72]: Then I see no reason to optimize now. Leave as-is.

**Review by BartleyR — APPROVED** [review]

**Review by raykallen — APPROVED** [review]

**Review by efajardo-nv — APPROVED** [review]

**Pull request merged**